Sources/MapboxDirections/RouteRefreshResponse.swift

```diff
@@ -99,9 +99,9 @@ extension Route {
      
      - parameter refreshedRoute: The route containing leg attributes to merge into the receiver. If this route contains fewer legs than the receiver, this method skips legs from the beginning of the route to make up the difference, so that merging the attributes from a one-leg route affects only the last leg of the receiver.
      */
-    public func refreshLegAttributes(from refreshedRoute: RefreshedRoute) {
-        for (leg, refreshedLeg) in zip(legs.suffix(refreshedRoute.legs.count), refreshedRoute.legs) {
-            leg.attributes = refreshedLeg.attributes
+    public func refreshLegAttributes(from refreshedRoute: RouteRefreshing) {
+        for (leg, refreshedLeg) in zip(legs.suffix(refreshedRoute.refreshedLegs.count), refreshedRoute.refreshedLegs) {
```

Comment:
Should we handle the case where `refreshedRoute` has more legs than the current one?

Reply:
This should never happen in real life. But in such case it would just skip excessive legs as it does when there are fewer legs